oauthlib/oauth1/rfc5849/endpoints/signature_only.py

```diff
@@ -34,13 +34,17 @@ def validate_request(self, uri, http_method='GET',
         """
         try:
             request = self._create_request(uri, http_method, body, headers)
-        except errors.OAuth1Error:
+        except errors.OAuth1Error as e:
```

Comment:
Sure -- I was following the convention(?) in https://github.com/idan/oauthlib/blob/67f973ff7f98bb3d892a33eda67ba1dab3bddead/oauthlib/oauth2/rfc6749/endpoints/base.py#L58 though -- down to the logging message.